LEAF_Request_Portal/admin/templates/mod_conditions_editor.tpl

```diff
@@ -0,0 +1,12 @@
+<div id="LEAF_conditions_editor"></div>  <!-- vue mount -->
+
+<script src="https://unpkg.com/vue@3"></script>

```

Comment:
Libraries should be hosted locally in ./libs.

Reply:
We have pushed VUE to libs and removed the external link cdn.